### DISCUSSION TITLE
why are the players who ruin the game playing on and I'm banned for 24 hours

### DISCUSSION THREAD

**Discussion by Sm1leFriend** [ValveSoftware/Dota2-Gameplay#16896] [General]
why are the players who ruin the game playing on and I'm banned for 24 hours
The hard-carry broke the items and died on purpose
Semi-support blocked me in the fight so that I would die
why was I banned and the players who really ruined the game keep playing
match id 7709562343

**Comment by Espionage724** [comment]
Because your behavior was (most likely) determined and picked-out by automation or AI. Presumably there's more to the story, but there's no way reports, bad-behavior, or moderator actions are monitored by a real person most of the time outside of extraordinary circumstances.

AI has no capability of judging the whole picture. AI does what it does in a straightforward nonchalant manner; you get X amounts of reports in X amount of time, possibly a sprinkle of whatever coders call "whole picture empathy" (unless that's reserved for big-time players), and temp ban. X temp bans = perma ban.

Dota 2 is a moving train of hundreds or thousands of games going on every hour and doesn't have time to have a human analyze every game, every report, the whole picture of voice and text chat, and to understand the in-game mechanics that led up to incidents, along with employing a person who isn't just going to act like AI and ban no-questions-asked when nobody's looking. That would be fair player moderation, expensive, and probably a large waste on the assumption most people don't do target others with large-scale bulk reports.

(I know nothing about actual moderation on Dota 2 and am giving my opinion of what I think happened)
